.github/workflows/go.yml

```diff
@@ -14,7 +14,55 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ matrix.go-version }}
-    - name: Checkout code
+    - name: Checkout code to gopath
       uses: actions/checkout@v1
-    - name: Build & Test
-      run: make
\ No newline at end of file
+      with:
+        path: go/src/github.com/hyperledger-labs/fabric-chaincode-wasm
+    - name: Debug dirs and env
+      run: |
+        pwd
+        echo "###### ENV #######"
+        env
+        echo "###### GO ENV #######"
+        go env
+        echo ${GOPATH}
```

Comment:
Is this piece necessary for ci?

Reply:
Nope, just added it for more logs. I will remove it